app/src/androidTest/java/com/example/mylittlelibrary/ExampleInstrumentedTest.kt

```diff
@@ -10,7 +10,6 @@ import org.junit.Assert.*
 
 /**
  * Instrumented test, which will execute on an Android device.
- *
```

Comment:
Empty line removed from multi-line comment to better match the example on kotlinlang.org. https://kotlinlang.org/docs/coding-conventions.html#documentation-comments